src/mpyl/steps/deploy/kubernetes.py

```diff
@@ -46,6 +46,28 @@ def try_extract_hostname(
                 return DeployKubernetes.match_to_url(match)
         return None
 
+    @staticmethod
+    def get_endpoint(builder: ChartBuilder) -> str:
+        step_input = builder.step_input
+        has_specific_routes_configured: bool = bool(
```

Comment:
Did you guys get errors, or why add the explicit type & typecasting? It's not needed afaik, the inferred type is already `bool`

Reply:
that code was already there, I only moved it

The type is inferred indeed, removed that 🧹 if I remove the typecasting as well, the inferred type becomes `Any | Literal[False]`. Unclear if it's something related to my setup, because running `pipenv run validate` seems fine locally 🤔

Reply:
Right of course, sorry! 

I tried it myself and for me it does just show that the type is `bool`. I think we also do this in more places, but it's hard to search for it. I can't screenshot the inferred type mouseover, but here's the normal one:

<img width="394" alt="Screenshot 2024-01-23 at 21 01 00" src="https://github.com/Vandebron/mpyl/assets/56846255/51bc59f2-559e-40b0-8509-cd1d6451de57">

Reply:
hmm then it must be something on my side. Updated, removed this typecasting 🧹